<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>631</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_titre">
      <property name="font">
       <font>
        <family>Candara</family>
        <pointsize>27</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Calcul de temps de route</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="margin">
       <number>6</number>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_diametre">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_diametre">
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Candara</family>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Diamètre :</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_diametre_result">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_aide">
        <property name="title">
         <string>Aide</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_aide">
           <property name="font">
            <font>
             <family>Candara</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string> - Cliquez sur Options&gt;Réinitialiser afin de revenir au contenu de la page par défaut.
 - Sélectionnez des villes des liste &quot;Départ&quot; et &quot;Arrivée&quot; afin de calculer les temps
de route ainsi que les valeurs du tableau.</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="layout_contenu">
      <item>
       <widget class="QGroupBox" name="groupBox_dep_arr_temps">
        <property name="title">
         <string>Temps entre deux villes</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="layout_titres_dep_arr">
           <item>
            <widget class="QLabel" name="label_depart">
             <property name="font">
              <font>
               <family>Candara</family>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Départ</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_arrivee">
             <property name="font">
              <font>
               <family>Candara</family>
               <pointsize>13</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Arrivée</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="margin">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout_dep_arr">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <widget class="QListWidget" name="depart">
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListWidget" name="arrivee">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>250</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="layout_temps">
           <item>
            <widget class="QLabel" name="label_temps">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Candara</family>
               <pointsize>11</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Temps :</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_temps_res">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="buttonTemps">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Calculer le temps de route</string>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_tableau">
        <property name="title">
         <string>Tableau de temps</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_tableau">
           <property name="font">
            <font>
             <family>Candara</family>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Temps</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableau">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonTab">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>35</height>
            </size>
           </property>
           <property name="text">
            <string>Mettre à jour le tableau</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1165</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionA_propos"/>
    <addaction name="actionReinitialiser"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionA_propos">
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionReinitialiser">
   <property name="text">
    <string>Réinitialiser</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
